src/components/OurOffer/helpers.ts

```diff
@@ -0,0 +1,25 @@
+export interface IPlan {
```

Comment:
Since this file contains interfaces, calling it helpers seems a bit odd. I'd suggest renaming it to something like interfaces. 🙂